Vikunja Update fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Vikunja

### 📂 What was the exact command used to execute the script?

update or bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/vikunja.sh)"

### 📝 Provide a clear and concise description of the issue.

When trying to update Vikunja from 24.5 to 24.6, it failed and folder `/opt`  is now empty..
See error output ⬇️


### ⚙️ What settings are you using?

- [ ] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 🔄 Steps to reproduce the issue.

try `update` or `bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/vikunja.sh)"` on Vikunja 24.5

### ❌ Paste the full error output (if available).

```
  ✔️  Stopped Vikunja
⠸ Updating Vikunja to 0.24.6environment: line 40: DEBIAN_FRONTEND=noninteractive: command not found
⠼ Updating Vikunja to 0.24.6
[ERROR] in line 40: exit code 0: while executing command $STD DEBIAN_FRONTEND=noninteractive dpkg -i vikunja-$RELEASE-amd64.deb
```


### 🖼️ Additional context (optional).

to fix, i have to manually install:
```
# RELEASE=$(curl -s https://dl.vikunja.io/vikunja/ | grep -oP 'href="/vikunja/\K[0-9]+\.[0-9]+\.[0-9]+' | sort -V | tail -n 1)
# wget -q "https://dl.vikunja.io/vikunja/$RELEASE/vikunja-$RELEASE-amd64.deb"
# dpkg -i vikunja-$RELEASE-amd64.deb
# systemctl start vikunja.service
```

Now vikunja works again.. and is updated to 24.6

--- a/ct/vikunja.sh
+++ b/ct/vikunja.sh
@@ -37,7 +37,8 @@ function update_script() {
     cd /opt
     rm -rf /opt/vikunja/vikunja
     wget -q "https://dl.vikunja.io/vikunja/$RELEASE/vikunja-$RELEASE-amd64.deb"
-    $STD DEBIAN_FRONTEND=noninteractive dpkg -i vikunja-$RELEASE-amd64.deb
+    export DEBIAN_FRONTEND=noninteractive
+    $STD dpkg -i vikunja-$RELEASE-amd64.deb
     echo "${RELEASE}" >/opt/${APP}_version.txt
     msg_ok "Updated ${APP}"
 

